--- vendor/php-stubs/wp-cli-stubs/wp-cli-commands-stubs.php	2024-03-26 10:54:38
+++ vendor/php-stubs/wp-cli-stubs/wp-cli-commands-stubs-copy.php	2024-03-26 10:54:35
@@ -10773,6 +10773,7 @@
         public function __construct($iterator, $callback)
         {
         }
+        #[\ReturnTypeWillChange]
         public function current()
         {
         }
\ No newline at end of file
@@ -10786,18 +10787,23 @@
         public function __construct($post_ids, $limit = \null)
         {
         }
+        #[\ReturnTypeWillChange]
         public function current()
         {
         }
+        #[\ReturnTypeWillChange]
         public function key()
         {
         }
+        #[\ReturnTypeWillChange]
         public function next()
         {
         }
+        #[\ReturnTypeWillChange]
         public function rewind()
         {
         }
+        #[\ReturnTypeWillChange]
         public function valid()
         {
         }
\ No newline at end of file
