--- vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php	2024-03-26 10:49:07
+++ vendor/php-stubs/wp-cli-stubs/wp-cli-stubs-copy.php	2024-03-26 10:50:21
@@ -2460,21 +2460,27 @@
         public function __construct($filename, $delimiter = ',')
         {
         }
+        #[\ReturnTypeWillChange]
         public function rewind()
         {
         }
+        #[\ReturnTypeWillChange]
         public function current()
         {
         }
+        #[\ReturnTypeWillChange]
         public function key()
         {
         }
+        #[\ReturnTypeWillChange]
         public function next()
         {
         }
+        #[\ReturnTypeWillChange]
         public function count()
         {
         }
+        #[\ReturnTypeWillChange]
         public function valid()
         {
         }
